roles/common/postgres/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Install postgresql
+  apt:
+    name: postgresql
+    state: present
+    update_cache: yes
+
+- name: Install psycopg2
```

Comment:
This looks like a very specific thing. Is it an ansible requirement? If so, a comment would be useful.